backends/dpdk/backend.cpp

```diff
@@ -52,10 +52,12 @@ void DpdkBackend::convert(const IR::ToplevelBlock* tlb) {
 
     PassManager simplify = {
         new DpdkArchFirst(),
+        new ByteAlignment(typeMap, refMap, &structure),
```

Comment:
I think ByteAlignment is a passmanager which does typechecking as well. Please check once

Reply:
yes it's there, will remove it